src/content/docs/telemetry-data-platform/ingest-manage-data/manage-data/query-limits.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Query system limits
+tags:
+  - Telemetry Data Platform
+  - Ingest data manage data
+  - Manage data
+  - Resource metrics
+  - system limits
+
+---
+
+As you use the New Relic system there are resource limits in place to protect your experience, our systems, and our other customers. These limits range from the maximum number of characters you can have in a query, to API request rates, to how many events your queries inspect. And everywhere in between.
+
+This page describes the limit metrics and NrIntegration events that enable you to view  your limits, your current data usage and overall resource consumption as compared to those limits, and the impact of experiencing a limit event. We also provide a handful of queries that, when compiled into a dashboard, can give you consistent insight into your limits status.
+
+<Callout variant="important">
+While `NrIntegration` events cover nearly all limits, resource metrics currently only cover request rate ingestion limits.
```

Comment:
`NrIntegration` -> `NrIntegrationError`

Reply:
I would say While `NrIntegrationError` events cover many limits, resource . . . 

I still think there are many limits that we do not cover with NrIntegrationErrors.